aea/aea_builder.py

```diff
@@ -1240,10 +1209,23 @@ def _load_connection(
             return cast(
                 Connection,
                 load_component_from_config(
-                    ComponentType.CONNECTION, configuration, address=address
+                    ComponentType.CONNECTION,
+                    configuration,
+                    identity=identity,
+                    cryptos=wallet.connection_cryptos,
                 ),
             )
 
+    def _load_and_add_connections(
+        self,
+        aea: AEA,
+        wallet: Wallet,
+        connection_ids: Optional[Collection[PublicId]] = None,
+    ):
+        connections = self._load_connections(aea.identity, wallet, connection_ids)
+        for c in connections:
+            aea.multiplexer.add_connection(c, c.public_id == self._default_connection)
```

Comment:
@DavidMinarsch it is ok for now, but the connection could be:

- added to `resources` at build time
- retrieved from resources and added to the multiplexer on start